src/templates/fr/2019/base.html

```diff
@@ -0,0 +1,166 @@
+        <a href="{{ url_for('contributors', year=year, lang=lang) }}">Contributeurs et contributrices</a>
+        <a href="{{ url_for('methodology', year=year, lang=lang) }}">Méthodologie</a>
+        {% if supported_years | length > 1 %}
+          {{ year_switcher() }}
+        {% endif %}
+      </nav>
+      <div class="cta">
+        {% if request.endpoint == 'home' %}
+          <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}" class="alt btn">
+            Commencer l'exploration
+          </a>
+        {% endif %}
+        {{ language_switcher() }}
+      </div>
+      <button type="button" class="menu-btn" arial-label="Menu">
+        <img class="menu-icon" src="/static/images/menu.png" data-src-close="/static/images/close.png" alt="" />
+      </button>
+      <nav class="menu">
+        <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">Table des matières</a>
+        <a href="{{ url_for('contributors', year=year, lang=lang) }}">Contributeurs et contributrices</a>
+        <a href="{{ url_for('methodology', year=year, lang=lang) }}">Méthodologie</a>
+        <div class="misc">
+          <a href="https://httparchive.org/">
+            <img src="/static/images/ha-home.svg" alt="Page d'accueil HTTP Archive" height="35" width="70">
+          </a>
+          <div class="social-media">
+            <a href="https://twitter.com/HTTPArchive" class="twitter">
+              <img src="/static/images/twitter.png" height="80" width="65" alt="Twitter" loading="lazy" />
+            </a>
+            <a href="https://github.com/HTTPArchive/almanac.httparchive.org" class="github">
+              <img src="/static/images/github.png" height="80" width="78" alt="GitHub" loading="lazy" />
+            </a>
+          </div>
+          {{ language_switcher() }}
+        </div>
+      </nav>
+    </div>
+    {% if request.endpoint == 'home' %}
+    <div class="intro-container">
+      <div class="intro">
+        <div class="intro-year">2019</div>
+        <h1 class="title title-lg title-alt">Web Almanac</h1>
+        <h2>
+          Rapport annuel<br>
+          de HTTP Archive sur<br>
+          <b>l'état du Web</b>
+        </h2>
+        <p>
+          Notre mission est de combiner les statistiques brutes et les tendances de HTTP Archive avec l'expertise de la communauté web. Le Web Almanac est un rapport complet sur l'état du Web, soutenu par des données réelles et des experts du Web. Il est composé de 20 chapitres couvrant l'expérience utilisateur, le contenu des pages, leur publication et leur distribution.
+        </p>
+        <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}" class="btn">
+          Commencer l'exploration
+        </a>
+      </div>
+      <img class="intro-image-2019" src="/static/images/home-hero-2019.png" alt="The Web Almanac 2019 edition" />
+      <img class="intro-image" src="/static/images/home-hero.png" alt="The Web Almanac 2019 edition" />
```

Comment:
Any translation needed on these two alt attributes?

Reply:
Go on! You've done some coding here 😀Plus might need some more from you before this translation's done 😉

```suggestion
        "developers",
        "translators"
```